conda/build.sh

```diff
@@ -28,5 +32,20 @@ ninja -v all-tests install
 ./dataset/test/scipp-dataset-test
 ./neutron/test/scipp-neutron-test
 
+# Build and run benchmarks if they were requested
```

Comment:
I suppose an intermediate solution would be to just build them (prevent further broken things), but avoid running, to save the time and memory.